src/lib/logger.js

```diff
@@ -42,39 +42,63 @@ function getDate(): string {
   return (new Date()).toISOString();
 }
 
-class Logger {
-  static container: Container;
 
-  static log(...params: Array<any>) {
+export class Logger {
```

Comment:
Move this to its own file and call it `DefaultLogger`

It would also be good to add a `ILogger` interface (in types.js):
```
export interface ILogger {
  error(params: Array<any>): void,
  info(params: Array<any>): void,
  log(params: Array<any>): void,
  warn(params: Array<any>): void,
}
```